src/rules/newline-after-import.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @fileoverview Rule to enforce new line after import not followed by another import.
+ * @author Radek Benkel
+ */
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+function getLineDifference(node, nextToken) {
+  return nextToken.loc.start.line - node.loc.start.line
+}
+
+function ensureNoForbiddenKeyword(context, node, tokenToInspect, tokenValue) {
+  if (getLineDifference(node, tokenToInspect) === 1
+    && tokenToInspect.type === 'Keyword' && tokenToInspect.value !== tokenValue)
```

Comment:
My bad, but I have a hard time understanding the name `ensureNoForbiddenKeyword` (never did any rules that dealt with tokens), and how/what exactly it it testing (except the line difference thing, that I got). Do you mind explaining it to me?